packages/flutter_markdown/pubspec.yaml

```diff
@@ -13,7 +14,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  markdown: ^5.0.0
+  markdown: #^5.0.0
```

Comment:
This is problematic. We are going to need a published version of `markdown` that we can use as the minimum supported version here. 

@devoncarew thoughts?

Reply:
i updated the commented version here to reference `6.0.0-dev`, which is the current version listed in the `pubspec.yaml` of the markdown repo. 
This is still commented out until the markdown is published to pub.dev, and instead has the reference directly to the github repo.  I added a comment to this affect.